executive-director.md

```diff
@@ -21,7 +21,8 @@ mission.
 * Responsible for communicating effectively with the Steering
   Committee and providing, in a timely and accurate manner, all
   information necessary for the Steering Committee to function
-  properly and to make informed decisions.
+  properly and to make informed decisions.  This information includes
+  reports of any executive actions.
```

Comment:
On Wed, May 11, 2016 at 02:04:29PM -0700, Raniere Silva wrote:
> > -  properly and to make informed decisions.
> > +  properly and to make informed decisions.  This information includes
> > +  reports of any executive actions.
> 
> This looks good to me. @jduckles already does it. :-)

Yeah, I imagine it's pretty obvious, but it doesn't hurt to be
explicit ;).

My only remaining concern would be “how granular is this reporting?”.
For example, in light of #111 and #114, I expect “I flipped a mailing
list moderator bit” and “I banned $PERSON from $CHANNEL” should be
reported, and not “I'm continuing to maintain $CHANNEL”.  In this
case, [1] is pretty close to my ideal with both the action (refusing a
discuss@ message) and a rationale (discuss@ is too noisy right now).
But I'm fine leaving it up to the ED and SC to work out the
appropriate level of detail.

[1]: https://github.com/swcarpentry/board/issues/111#issuecomment-217494630